templates/zerver/help/reply-to-message.md

```diff
@@ -0,0 +1,58 @@
+# Replying to a Message
+There are three ways to reply to a message in Zulip.
+## Clicking on a message
+To reply to a message, click on the message.
+A message compose box will appear with the **Stream** (outlined in red below)
+and **Topic** (outlined in green below) fields
+autofilled to match the message you clicked on.
+
+![reply messaging box](/static/images/help/reply-messaging-box.png)
+
+You can type your reply in the **Compose your message here...** field.
+Learn about formatting messages [here](/help/format-your-message-using-markdown).
+
+## The chevron icon
+Hover over the message that you want to reply to.
+Three icons will appear at the far right side of the message.
+
+![hover icons](/static/images/help/message-hover.png)
+
+Click on the chevron (![chevron](/static/images/help/chevron.png)) icon.
+This menu will appear:
+
+![menu](/static/images/help/menu-chevron-message.png)
```

Comment:
This should be consistent with the previous one and show a compose box with the message you've clicked the chevron icon on visible.